packages/core/src/MapIt/index.js

```diff
@@ -333,6 +335,23 @@ function MapIt({
       }
     };
 
+    /* eslint-disable global-require */
+    if (marker) {
+      const locationIcon = leaflet.icon({
+        iconUrl: require("../assets/icons/pointer.svg"),
+        iconRetinaUrl: require("../assets/icons/pointer.svg"),
```

Comment:
Any reason why we're not importing this as we do with any other resources i.e. `import iconUrl from '../assets/...'` at the top of the file?

Reply:
@kilemensi For some reason the marker image doesn't pick up with imports. 👇  using it the leaflet way seems to work
<img width="353" alt="Screenshot 2020-11-25 at 19 35 25" src="https://user-images.githubusercontent.com/3693563/100256262-a4e30880-2f55-11eb-9ae4-5abb40e4e388.png">

Reply:
Yes, it's a known issue with our Storybook setup i.e. not your code. It will work with Nextjs, etc., since they have better svg loaders.

_But_, it might also be nice to allow someone to specify which icon image they want to use via a `prop` and use this one as default if no icon is specified.

Reply:
@nerdlet Okay, on it